### ISSUE TITLE
Rewards are always [-inf]

### ISSUE THREAD

**Issue by edgarcamilocamacho** [beyretb/AnimalAI-Olympics#7]
I am testing with the following code *testMovement.py*:

``` python
from animalai.envs.environment import UnityEnvironment
from animalai.envs.ArenaConfig import ArenaConfig
import matplotlib.pyplot as plt

env= UnityEnvironment(
    file_name='env/AnimalAI',   # Path to the environment
    worker_id=6,                # Unique ID for running the environment (used for connection)
    seed=0,                     # The random seed
    docker_training=False,      # Whether or not you are training inside a docker
    no_graphics=False,          # Always set to False
    n_arenas=1,                 # Number of arenas in your environment
    play=False                  # Set to False for training
)

arena_config_in = ArenaConfig('configs/justFood.yaml')
env.reset(config=arena_config_in, train_mode=True)

plt.ion()

def step(action):
    info = env.step(action)
    b1 = info['Learner']
    img = b1.visual_observations
    plt.imshow(img[0][0])
    print(f'Rewards: {b1.rewards}')
```

And I execute it in interactive mode:

``` bash
$ python -i testMovement.py
```

Then, I send actions manually, but I always get `rewards=[-inf]` (even if I reach the green food):

``` python
>>> step([0,0])
Rewards: [-inf]
>>> step([1,0])
Rewards: [-inf]
>>> step([2,0])
Rewards: [-inf]
>>> step([1,1])
Rewards: [-inf]
>>> step([1,2])
Rewards: [-inf]
>>> step([1,0])
Rewards: [-inf]
>>> step([2,0])
Rewards: [-inf]
```

Changing the reset line to:

``` python 
env.reset(config=arena_config_in, train_mode=False)
```

I get the same rewards, but it is easier to test.

**Comment by beyretb** [comment]
Hi,

Thank you for reporting this issue. This comes from the fact that you run the environment with no time limit (`t: 0` in the configuration file) which should return a reward of 0 unless you get to a goal (reward of 1).

I will fix this in the v0.2 which will come this weekend, in the meantime you can set `t: 1000` to obtain the rewards using your script.

**Comment by beyretb** [comment]
Closing as v0.2 fixes the issue
